cob_twist_controller/cfg/TwistController.cfg

```diff
@@ -91,7 +91,7 @@ limits = gen.add_group("Limits", "limits")
 limits.add("keep_direction",     bool_t,   0, "With keep_direction the whole joint positions and velocities vector is affected by a scaling factor. Else only individual components of the vectors are affected -> direction will be changed.", True)
 limits.add("enforce_vel_limits", bool_t,   0, "If 'True', enforce joint velocity limits in TwistController, else let RobotHW handle the limits",  True)
 limits.add("enforce_pos_limits", bool_t,   0, "If 'True', enforce joint position limits in TwistController, else let RobotHW handle the limits",  True)
-limits.add("tolerance",          double_t, 0, "Tolerance around joint position limits where velocities are normalized [degrees]",  5, 0, 15)
+limits.add("tolerance",          double_t, 0, "Tolerance around joint position limits where velocities are normalized [degrees]. Must not be zero else DIV/0!",  5, 0.001, 15)
```

Comment:
Added: 


    if(twist_controller_params_.tolerance <= DIV0_SAFE)
    {
        ROS_ERROR("The tolerance for enforce limits is smaller than DIV/0 threshold. Therefore both enforce_limits are set to false!");
        twist_controller_params_.enforce_pos_limits = config.enforce_pos_limits = false;
        twist_controller_params_.enforce_vel_limits = config.enforce_vel_limits = false;
    }